build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -40,6 +40,8 @@
         name: amq7-cert-manager-operator
         source: redhat-operators
         sourceNamespace: openshift-marketplace
+  when:
+    - __service_telemetry_observability_strategy == "use_community"
```

Comment:
I think this is wrong. The AMQ Interconnect Operator uses AMQ Cert Manager to create the certificates required for the routes that OpenStack connects to. I likely need to just remove this line, or add an `or .... == "none"`. In the future we may add `use_community_sub` and `use_openshift` parameters. I don't want to fully drop the `when` clause here just in case we forget to adjust it.

Reply:
Oh, yes, I agree. Good catch!